wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
@JonWel , in SMath wrapper, I use buffer of 10 000 chars long.
I don't think limiting it too much is a good idea, because e.g. fluids list is constantly growing, and it may be required to constantly monitor this.